.github/settings.yml

```diff
@@ -111,11 +111,6 @@ branches:
           - sort
           - "clippy (stable)"
           - "build (stable)"
```

Comment:
Got rid of the no alloc. While it sounded nice, the workaround for it was extra work that we probably should take on at this time.